site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -461,7 +487,59 @@ components:
          description: Allows arbitrary properties as extension to the ones
                       specified by the Hono API.
 
-# Tenant schema
+
+      PagingSortingFilteringOptions:
+         type: object
+         additionalProperties: false
+         properties:
+            "page":
+               $ref: '#/components/schemas/PagingOptions'
+            "filters":
+               $ref: '#/components/schemas/FilterOptions'
+            "sort":
+               $ref: '#/components/schemas/SortingOptions'
+
+      PagingOptions:
+         type: object
+         additionalProperties: false
+         description: options to page the results of bulk queries.
+         properties:
+            "limit":
+               type: integer
+               default: 30
+               description: |
+                  The threshold on the number of results returned in a single API call. If not set, the registry will return up to 30 results.
+                  The upper bound for this value is 200 and it cannot be less than 0.
```

Comment:
we should be able to formally define the min and max using the `minimum` and `maximum` properties ...